patches/0012-Disabling-Mozilla-Tracking-protection.patch

```diff
@@ -1,64 +1,64 @@
-From f0137dcd9fedc47784009553efbbeb90734ef42d Mon Sep 17 00:00:00 2001
+From 802411ca0de5a3564f465d515466ff8f4ed3d9f6 Mon Sep 17 00:00:00 2001
```

Comment:
Would it make sense to keep Mozilla's tracking protection enabled? Might mess up the visualization and antitracking/WTM related stats though.

Reply:
AFAICT Ghostery provides stronger protection. No need to run both.